packages/lab/src/ImageCarousel/tests/ImageCarousel.test.js

```diff
@@ -0,0 +1,140 @@
+/* eslint-env jest */
+
+import React from "react";
+
+import { render, screen, fireEvent } from "testing-utils";
+
+import {
+  Main,
+  WithoutThumbnails,
+  LowCardinality,
+  ImageCarouselXS,
+} from "../stories/ImageCarousel.stories";
+
+describe("<HvImageCarousel>", () => {
+  describe("sample snapshot testing", () => {
+    xit("Main", () => {
+      const { container } = render(<Main />);
+      expect(container).toMatchSnapshot();
+    });
+    xit("WithoutThumbnails", () => {
+      const { container } = render(<WithoutThumbnails />);
+      expect(container).toMatchSnapshot();
+    });
+    xit("LowCardinality", () => {
+      const { container } = render(<LowCardinality />);
+      expect(container).toMatchSnapshot();
+    });
+    xit("ImageCarouselXS", () => {
+      const { container } = render(<ImageCarouselXS />);
+      expect(container).toMatchSnapshot();
+    });
+  });
```

Comment:
These xit functions are leftovers?, if leftovers enable the tests again, if not erase them